### PR TITLE
split TM & PL for Bolts and Flash

### PR DESCRIPTION
**Pull request by Borda**
## Before submitting

- [ ] Was this discussed/approved via a GitHub issue? (no need for typos and docs improvements)
- [x] Did you create/update your **configuration file**?
- [x] Did you **set `runtimes` in config** for GitHub action integration?
- [ ] Did you **add your config to CI** in Azure pipeline (only projects with 100+ GitHub stars)?
- [ ] Are all integration **tests passing**?

## What does this PR do? \[optional\]

split testing for latest TM and PL in Bolts and Flash

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
pending on https://github.com/Lightning-AI/lightning-flash/pull/1504

File: configs/Lightning-AI/bolts_tm-release.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  checkout: master
+  # define installing package extras
+  install_extras: extra
+  # copy some  test from the target repository
+  copy_tests:
+    - tests
+
+contact:
+  slack:
+    # Rohit Gupta
```
Comment (justusschock) [inline configs/Lightning-AI/bolts_tm-release.yaml:13]: Update Point of contact here?

**Review by justusschock — APPROVED** [review]

File: configs/Lightning-AI/bolts_tm-release.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    checkout: release/stable
+    # install_extras: all
+
+testing:
+  dirs:
+    - tests
+
+runtimes:
+  - {os: "ubuntu-20.04", python: "3.8"}
+  - {os: "macOS-10.15", python: "3.8"}
```
[the remark this answers is not in the document]
Comment (Borda) [reply configs/Lightning-AI/bolts_tm-release.yaml:28]: just to keep it light and from past experience, what worked on Unix worked on Win

[the remark this answers is not in the document]
Comment (justusschock) [reply configs/Lightning-AI/bolts_tm-release.yaml:28]: I'd rather have testing on unix and win than on unix and mac (which are much more similar) :) Windows is different in a number of ways which means that we should definitely test there :)

**Review by akihironitta — APPROVED** [review]

**Pull request closed without merging**